tests/CMakeLists.txt

```diff
@@ -199,6 +199,7 @@ if(AVIF_ENABLE_FUZZTEST)
                 "fuzztest: Installed FuzzTest is not supported, please set AVIF_LOCAL_FUZZTEST=ON or AVIF_ENABLE_FUZZTEST=OFF"
         )
     endif()
+    fuzztest_setup_fuzzing_flags()
```

Comment:
Just curious: did you write this function before and it was not used until now?

Reply:
This is actually part of the fuzztest framework and is part of the doc https://github.com/google/fuzztest/blob/main/doc/quickstart-cmake.md. It actually helps for ubsan tests that also require to be linked to asan but was not necessary for the main tests I cared about so far (pure asan tests). Let's get them all now :)